kythe/go/extractors/config/BUILD

```diff
@@ -8,6 +8,9 @@ go_library(
         "config_parser.go",
         "extractor.go",
     ],
+    data = [
+        "//kythe/java/com/google/devtools/kythe/extractors/java/artifacts",
```

Comment:
Buildbot doesn't currently support `docker` so this rule will need to be tagged `"manual"`.

Reply:
Hmm.  So then we have to figure out if we want to make the whole transitive stack up for this manual or not.

Probably not.

Reply:
I'll just close this for now then.